multimedia/yt-dlp/Makefile

```diff
@@ -1,16 +1,19 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=yt-dlp
-PKG_VERSION:=2023.12.30
+PKG_VERSION:=2024.5.27
 PKG_RELEASE:=1
 
 PYPI_NAME:=yt-dlp
-PKG_HASH:=a11862e57721b0a0f0883dfeb5a4d79ba213a2d4c45e1880e9fd70f8e6570c38
+PKG_HASH:=3566c0de240d0cd3d1c2285ce655f72ca38dfc618d634d46818b00d89d5288be
+PYPI_SOURCE_NAME:=yt_dlp
 
 PKG_MAINTAINER:=Michal Vasilek <michal.vasilek@nic.cz>
 PKG_LICENSE:=Unlicense
 PKG_LICENSE_FILES:=LICENSE
 
+PKG_BUILD_DEPENDS:=python-hatchling/host
```

Comment:
This was added and it is not said anywhere why.

Reply:
It's required to build this package. Should I do `git commit --amend`?

Reply:
Sure

Reply:
done